lib/inflector.php

```diff
@@ -238,7 +238,7 @@ public function camelize($term, $downcase_first_letter = self::UPCASE_FIRST_LETT
 
 				$m = $matches[0];
 
-				return !empty($acronyms[$m]) ? $acronyms[$m] : capitalize($m);
+				return !empty($acronyms[$m]) ? $acronyms[$m] : ucfirst($m);
```

Comment:
`ucfirst()` won't do because it doesn't work with UTF-8. Maybe add a private method:

```php
static private function ucfirst($str)
{
    return mb_strtoupper(mb_substr($str, 0, 1)) . mb_substr($str, 1);
}
```